app/views/users/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h3> Your Events </h3>
+<p>
+   <ul id = "users">
+      <% @user.created_events.each do |event|%>
+      <li>
+          Event: <%= event.name %> <br/> 
+      </li>
+      <li>
+        Location: <%= event.location %><br/>
+      </li>
+      <li>     
+         Event_Date: <%= event.start_date %>
+      </li>
+      <%end%>    
+   </ul> 
+</p>
+<h3>Future Events</h3>
+<ul>
+   <%@user.attended_events.each do |event| %>
+      <% if Date.today < event.start_date%>
+      <%=event.eventname%>
+      <%end%>
+   <%end%>
+</ul>
+
+<h3>Past Events</h3>
+<ul>
+   <%@user.attended_events.each do |event| %>
```

Comment:
- [ ] Like above, this should also be in the users controller. 
Screenshot from requirements 👇 
<img width="826" alt="Screen Shot 2020-12-06 at 10 27 31 AM" src="https://user-images.githubusercontent.com/48804258/101284393-c772ee00-37ad-11eb-99c5-1d4db1d2dfe2.png">